packages/block-library/src/embed/index.js

```diff
@@ -210,28 +239,57 @@ export function getEmbedEdit( title, icon ) {
 				setAttributes( { type, providerNameSlug } );
 			}
 
-			this.maybeSetAspectRatioClassName( html );
+			this.setAspectRatioClassNames( html );
 		}
 
 		switchBackToURLInput() {
 			this.setState( { editingURL: true } );
 		}
 
+		getResponsiveHelp( checked ) {
+			return checked ? __( 'Videos and other content automatically resizes.' ) : __( 'Content is fixed size.' );
+		}
+
+		toggleResponsive() {
+			const { allowResponsive, className } = this.props.attributes;
+			const { html } = this.props.preview;
+			const responsiveClassNames = this.getAspectRatioClassNames( html, ! allowResponsive );
+
+			this.props.setAttributes(
+				{
+					allowResponsive: ! allowResponsive,
+					className: classnames( className, responsiveClassNames ),
+				}
+			);
+		}
+
 		render() {
 			const { url, editingURL } = this.state;
-			const { caption, type } = this.props.attributes;
+			const { caption, type, allowResponsive } = this.props.attributes;
 			const { fetching, setAttributes, isSelected, className, preview, previewIsFallback } = this.props;
 			const controls = (
-				<BlockControls>
-					<Toolbar>
-						{ ( preview && ! previewIsFallback && <IconButton
-							className="components-toolbar__control"
-							label={ __( 'Edit URL' ) }
-							icon="edit"
-							onClick={ this.switchBackToURLInput }
-						/> ) }
-					</Toolbar>
-				</BlockControls>
+				<Fragment>
+					<BlockControls>
+						<Toolbar>
+							{ ( preview && ! previewIsFallback && <IconButton
+								className="components-toolbar__control"
+								label={ __( 'Edit URL' ) }
+								icon="edit"
+								onClick={ this.switchBackToURLInput }
+							/> ) }
+						</Toolbar>
+					</BlockControls>
+					<InspectorControls>
+						<PanelBody title={ __( 'Media Settings' ) } className="blocks-responsive">
+							<ToggleControl
+								label={ __( 'Responsive' ) }
```

Comment:
This is still labelled as "Responsive", shouldn't the label change?

Reply:
Oooh, good catch! "Auto-fit content"? "Automatically scale content"?

Reply:
I like "Automatically scale content" 👍